cypress/integration/forms/date-inputs.spec.ts

```diff
@@ -1,20 +1,51 @@
 describe('Date/Time', () => {
   before(() => {
     cy.visit('/datetime');
+    cy.injectAxe();
     cy.get('.page-loader').should('not.exist', { timeout: 20000 });
   });
 
-  describe('Date Input', () => {
-    beforeEach(() => {
-      cy.get('ngx-date-time').first().as('CUT');
+  it('Check A11y', () => {
+    cy.get('ngx-date-time').withinEach($el => {
+      cy.checkA11y($el);
     });
+  });
 
-    afterEach(() => {
-      cy.get('@CUT').clear();
+  describe('Basic Date Input', () => {
+    beforeEach(() => {
+      cy.get('[sectiontitle="Date Input"]').as('SUT');
+      cy.get('@SUT').getByLabel('Date of attack').as('CUT');
+      cy.get('@SUT').getByLabel('Current Value:').as('output');
+      cy.get('@CUT').ngxFill('10/10/2016').ngxFindNativeInput().focus().blur();
     });
 
     it('has a label', () => {
-      cy.get('@CUT').ngxFindLabel().should('contain.text', 'Date of attack');
+      // With Value
+      cy.get('@CUT')
+        .ngxFindLabel()
+        .should('contain.text', 'Date of attack')
+        .should('have.css', 'color', 'rgb(129, 143, 169)')
```

Comment:
nit pick but it'd be more readable if we set these rgbs as const with descriptive names